<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateProfileDialog</class>
 <widget class="QDialog" name="UpdateProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x> <y>0</y> <width>400</width> <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update Profile</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QPushButton {
    background-color: #2e89ff;
    color: white;
    border-radius: 6px;
    padding: 6px 12px;
    font: 10pt "Segoe UI";
}
QPushButton:hover {
    background-color: #1c6edb;
}
QLabel, QLineEdit {
    font: 10pt "Segoe UI";
}
QLabel#cameraLabel {
    background-color: #f0f0f0;
    border: 1px solid #aaa;
    font: italic 9pt "Segoe UI";
}
   </string>
  </property>

  <layout class="QVBoxLayout" name="verticalLayout">

   <!-- Profile Update Section -->
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelFullName">
       <property name="text"><string>Full Name:</string></property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="fullNameLineEdit"/>
     </item>

     <item row="1" column="0">
      <widget class="QLabel" name="labelRole">
       <property name="text"><string>Role:</string></property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="roleLineEdit"/>
     </item>

     <item row="2" column="0">
      <widget class="QLabel" name="labelPassword">
       <property name="text"><string>New Password:</string></property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="passwordLineEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>

   <!-- Update Info Button -->
   <item>
    <widget class="QPushButton" name="updateInfoButton">
     <property name="text"><string>Update Info</string></property>
    </widget>
   </item>

   <!-- Camera Preview -->
   <item>
    <widget class="QLabel" name="cameraLabel">
     <property name="minimumSize">
      <size><width>320</width><height>240</height></size>
     </property>
     <property name="frameShape"><enum>QFrame::Box</enum></property>
     <property name="alignment"><set>Qt::AlignCenter</set></property>
     <property name="text"><string>Camera Preview</string></property>
    </widget>
   </item>

   <!-- Face Update Buttons -->
   <item>
    <layout class="QHBoxLayout" name="faceButtonsLayout">
     <item>
      <widget class="QPushButton" name="captureFaceButton">
       <property name="text"><string>Capture Face</string></property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateFaceButton">
       <property name="text"><string>Update Face</string></property>
      </widget>
     </item>
    </layout>
   </item>

   <!-- OK / Cancel Buttons -->
   <item>
    <layout class="QHBoxLayout" name="bottomButtonsLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation"><enum>Qt::Horizontal</enum></property>
       <property name="sizeHint" stdset="0">
        <size><width>40</width><height>20</height></size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBoxOk">
       <property name="text"><string>OK</string></property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBoxCancel">
       <property name="text"><string>Cancel</string></property>
      </widget>
     </item>
    </layout>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
